bokeh/charts/builder/bar_builder.py

```diff
@@ -36,12 +36,23 @@
 
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
-        xgrid=False, ygrid=True, **kw):
+        xgrid=False, ygrid=True, continuous_range=None, **kw):
+
+    if continuous_range and not isinstance(continuous_range, Range1d):
```

Comment:
OK, let's remove this because as you and @bryevdv said, the validation it is performed later...